kore/src/Kore/Log/Registry.hs

```diff
@@ -154,41 +163,48 @@ registry =
         (asText type', type')
 
 entryTypeReps :: [SomeTypeRep]
-entryHelpDocs :: [Pretty.Doc ()]
-(entryTypeReps, entryHelpDocs) =
-    unzip
-    [ mk $ Proxy @DebugSolverSend
-    , mk $ Proxy @DebugSolverRecv
-    , mk $ Proxy @DebugClaimState
-    , mk $ Proxy @DebugAppliedRewriteRules
-    , mk $ Proxy @DebugSubstitutionSimplifier
-    , mk $ Proxy @ErrorBottomTotalFunction
-    , mk $ Proxy @ErrorDecidePredicateUnknown
-    , mk $ Proxy @ErrorParse
-    , mk $ Proxy @ErrorVerify
-    , mk $ Proxy @ErrorRuleMergeDuplicateIds
-    , mk $ Proxy @ErrorRuleMergeDuplicateLabels
-    , mk $ Proxy @WarnFunctionWithoutEvaluators
-    , mk $ Proxy @WarnSymbolSMTRepresentation
-    , mk $ Proxy @WarnStuckClaimState
-    , mk $ Proxy @WarnIfLowProductivity
-    , mk $ Proxy @WarnTrivialClaim
-    , mk $ Proxy @WarnRetrySolverQuery
-    , mk $ Proxy @DebugEvaluateCondition
-    , mk $ Proxy @ErrorException
-    , mk $ Proxy @ErrorRewriteLoop
-    , mk $ Proxy @LogMessage
-    , mk $ Proxy @InfoAttemptUnification
-    , mk $ Proxy @InfoReachability
-    , mk $ Proxy @InfoExecBreadth
-    , mk $ Proxy @ErrorRewritesInstantiation
-    , mk $ Proxy @DebugAttemptEquation
-    , mk $ Proxy @DebugApplyEquation
-    , mk $ Proxy @DebugUnification
-    , mk $ Proxy @InfoProofDepth
-    , mk $ Proxy @InfoExecDepth
-    , mk $ Proxy @DebugProven
-    ]
+entryTypeReps = entryTypeRepsErr <> entryTypeRepsNoErr
+
+entryTypeRepsErr, entryTypeRepsNoErr :: [SomeTypeRep]
+entryHelpDocsErr, entryHelpDocsNoErr :: [Pretty.Doc ()]
+((entryTypeRepsNoErr, entryHelpDocsNoErr), (entryTypeRepsErr, entryHelpDocsErr))
```

Comment:
Could this be formatted on multiple lines? I think it's kind of long and hard to read.